src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/Bit.Client.Web.BlazorUI.Playground.Web.csproj

```diff
@@ -18,7 +18,7 @@
         <PackageReference Condition=" '$(BlazorMode)' == 'Client' " Include="Microsoft.AspNetCore.Components.WebAssembly" Version="6.0.0-rc.2.21480.10" />
         <PackageReference Condition=" '$(BlazorMode)' == 'Client' " Include="Microsoft.AspNetCore.Components.WebAssembly.DevServer" Version="6.0.0-rc.2.21480.10" PrivateAssets="all" />
     </ItemGroup>
-
+    
```

Comment:
no changes should be introduced in the project file in this PR.
plz copy the content of this file from the develop branch